variants/mkrnb1500/variant.cpp

```diff
@@ -252,9 +252,10 @@ void initVariant() {
   pinMode(SARA_PWR_ON, OUTPUT);
   digitalWrite(SARA_PWR_ON, LOW);
 
-  // put SARA modem in reset on start to conserve power if it's not used
+  // data sheet discourages use of SARA modem reset pin
+  // use may lead to irrecoverable state of modem
   pinMode(SARA_RESETN, OUTPUT);
-  digitalWrite(SARA_RESETN, HIGH);
+  digitalWrite(SARA_RESETN, LOW);
```

Comment:
Hi @janakelarsson :wave: This line is already in [ArduinoCore-samd:master:HEAD](https://github.com/arduino/ArduinoCore-samd/blame/0f502c7f193b0a51040c36eaacb36a333a5ad44d/variants/mkrnb1500/variant.cpp#L257) - please rebase your commit.

Reply:
Better now?

Reply:
Or do I need to squash the commits?

Reply:
Good now, I'll squash them myself.